### ISSUE TITLE
FNX-4995 ⁃ Settings categories are touch enabled

### ISSUE THREAD

**Issue by eeejay** [mozilla-mobile/fenix#1387]
Accessibility Scanner picks up the settings categories in the settings screen as having a too small touch target:

> Touch target
> [0,594][1080,687]
> This item's height is 35dp. Consider making the height of this touch target 48dp or larger.
> 
> Touch target
> [0,1536][1080,1629]
> This item's height is 35dp. Consider making the height of this touch target 48dp or larger.

The real issue seems to be that they are being picked up as interactive `uiautomatorviewer` shows them as long-clickable. And if you tap on them you see the highlight change. I don't think this should be the case. I can't find where this could be changed.

![image](https://user-images.githubusercontent.com/5764/55581551-0051ea00-56d2-11e9-840b-8aae04efaad9.png)

**Comment by ekager** [comment]
Good find! [This is a new Android bug introduced in the newest version of the Preference library.](https://issuetracker.google.com/issues/129835562)
For now, I've downgraded and commented on the bug so hopefully it will be fixed in the next version :)

**Comment by bsurd** [comment]
With accessibility enabled when trying to tap on them it no longer does any highlight to indicate that they are interactive.

I would like to note that the color for them has also been changed, it is now a darker color. @ekager was this intended with the change?

![screen](https://user-images.githubusercontent.com/17178945/55798395-7b9f0b80-5ad7-11e9-9c28-b75be605b146.png)
